### ISSUE TITLE
Error when commit or load state

### ISSUE THREAD

**Issue by ccqgithub** [vuejs/vue-devtools#980]
### Version
5.1.0

### Browser and OS info
chrome73 / macos 10

### Steps to reproduce
1. not open  vue devtool, my app is normal
2. open devtools, click `load state` or `commit`
3. always loading state, have error in console

```
VM120 backend.js:14674 Uncaught TypeError: Cannot read property 'config' of null
    at Store.setConfig (mutations.js:3)
    at wrappedMutationHandler (vuex.esm.js:714)
    at VM120 backend.js:14664
    at Array.forEach (<anonymous>)
    at VuexBackend.replayMutations (VM120 backend.js:14664)
    at VuexBackend.onInspectState (VM120 backend.js:14355)
    at Bridge.emit (VM120 backend.js:5472)
    at Bridge._emit (VM120 backend.js:5172)
    at VM120 backend.js:5097
    at Array.forEach (<anonymous>)
```

my mutation codes :
```js
// update config
export const setConfig = (state, confs = {}) => {
  state.config = { ...state.config, ...confs };
};
```

codes at error position:
```
function registerMutation (store, type, handler, local) {
  var entry = store._mutations[type] || (store._mutations[type] = []);
  entry.push(function wrappedMutationHandler (payload) {
    handler.call(store, local.state, payload);
  });
}
```

### What is expected?
Can be used normally

### What is actually happening?
don't know...

---
Previous versions are good. It seems that there is something wrong with updating to the latest version.

**Comment by kovaroman** [comment]
the same issue, click on `load state` and receive error:
```vuex.esm.js?2f62:753 Uncaught TypeError: Cannot read property 'settingsStore' of null
    at eval (vuex.esm.js?2f62:753)
    at Array.reduce (<anonymous>)
    at getNestedState (vuex.esm.js?2f62:753)
    at Object.get (vuex.esm.js?2f62:664)
    at wrappedMutationHandler (vuex.esm.js?2f62:697)
    at commitIterator (vuex.esm.js?2f62:389)
    at Array.forEach (<anonymous>)
    at eval (vuex.esm.js?2f62:388)
    at Store._withCommit (vuex.esm.js?2f62:495)
    at Store.commit (vuex.esm.js?2f62:387)
```
`settingsStore` is my Vuex module.

Browser Chrome 73.0.3683.103 x64, vue-devtools 5.1.0

**Comment by Akryum** [comment]
Could you provide a minimal runnable reproduction?

**Comment by depsimon** [comment]
I don't really know how to reproduce but I had this issue too.

By turning off new backend + autoload vuex state fixed it..

**Comment by pradeepb6** [comment]
I agree with @depsimon. When 'New Vuex Backend' is enabled, When clicked on 'Vuex' tab and tried to select mutation payload. It throws an error.  Disabling  'New Vuex Backend' fixed it though.

**Comment by yvescavalcanti** [comment]
Same problem, same solution. When i turn off "New Vuex Backend" the errors does not appear.

**Comment by gloomylumi** [comment]
I have been having the same issue. I have noticed it happens a lot after hot-reloading a Vue component with webpack dev server.

**Comment by kovaroman** [comment]
> Could you provide a minimal runnable reproduction?

I have found, that problem is with mutation payload, error is thrown from `/build/backend.js:14674`
At this point mutation has `payload = null`. All mutations in `this.mutations` have `payload = null`.

**Comment by ThomasKientz** [comment]
On my side, I had in my state an object with a key `toString` wich was causing the issue :

```
state() {
      return {
        foo: {
          toString: '',
        },
      }
    },
```

Renaming it to something else resolved the issue. My guess is that `toString` is also a JS method [.toString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/toString) and somehow this caused an issue with Vue DevTool (Chrome).

**Comment by badihi** [comment]
I agree with you @ThomasKientz, this problem happens when you have an object in the state that has a member called `toString`, that is not a function.

I think it is an issue that must be addressed ASAP.

**Comment by lesamelb-gwumkt** [comment]
i'm having the same issue, is there  any solution already that actually works??

**Comment by kbehlers** [comment]
@Akryum 
Runnable example to reproduce. The error is slightly different, but it may still lead to useful debugging as the stack trace closely mirrors the original reported error and the outcome of "always loading state, have error in console" is the same.
https://codepen.io/karldougie/pen/LYEWNjN
https://cdpn.io/karldougie/debug/LYEWNjN/XBAGRqNxgyRr
